src/css/base/table.scss

```diff
@@ -98,3 +112,20 @@ td table {
   margin: 0;
 }
 
+td .tableWrapper {
+  margin: 8px 0;
```

Comment:
I think this value should be in `rem`s because it's over 5px. Our rules stipulate that values under 5px can be in `px` and everything else should be in `rem`.